python/valis/db/db.py

```diff
@@ -17,7 +18,7 @@
 db_state = ContextVar("db_state", default=db_state_default.copy())
 
 
-async def reset_db_state():
+def reset_db_state():
     """ Sub-depdency for get_db that resets the context connection state """
```

Comment:
I think this change is ok since we're not using async yet anyway, peewee doesn't support it, and pydantic v2 deprecated their docs for this.  We can revisit any needed changes if we ever move to async db, which is still on my radar to explore for sqlalchemy.

Reply:
Yes, I think the sync is not needed here (and avoid having to make `get_pw_db` async as well). `reset_db_state` is not really blocking so I don't think there's a difference between it being sync or async.